2022/Day2/Day2.py

```diff
@@ -0,0 +1,44 @@
+file2 = open('input.txt', 'r')
+
+total_score = 0
+
+while True:
+    Line = file2.readline().strip()
+    if len(Line) == 0:
```

Comment:
This can be replaced with `if not line:` since empty lines are falsy.

Reply:
Will try to remember this in the future.